Portfolio website validate contact form:- module 1

yusufyusie opened the pull request:
### For this milestone of your portfolio website: validation check
**The following tasks will be included**

- Added content of the email field has to be in lowercase
- Added validation to show the status ok or not using inserted data
- Added error message if the user inserts data out of the data submission rule
- Updated the contact form design to keep that it must be same as the Figma design

akwuderaja commented on assets/js/formvalidate.js:
```diff
@@ -0,0 +1,19 @@
+    const email = document.getElementById('email').value;
+    const errorMsgElement = document.querySelector('.error_msg');
+    const errorMsg = 'Email should be in lower case.';
+    e.preventDefault();
+    const regex = /[A-Z]/;
+    if (!regex.test(email.value)) {
+      form.submit();
```
Consider replacing the whole of this section with the code below.

    const regex = /[A-Z]/;
    if (regex.test(email.value)) {
	e.preventDefault();

akwuderaja commented on index.html:
```diff
@@ -171,7 +169,7 @@ <h2 class="footer-title">I'm always interested in hearing about new projects, so
             <li class="submit-button">
               <button type="submit">Get in touch</button><br />
-              <small></small>
+              <div class="error_msg"></div>
```
Consider moving this line/element before the submit button. This is because keeping the error message after the submit button can somethings make it harder for the user to notice that something changed on the screen.

akwuderaja reviewed (commented):
Hi Yusuf, Almost there.
Kindly review the mentioned issues.

yusufyusie commented:
### For this milestone of your portfolio website: validation check
**The following tasks will be included**

- Added content of the email field has to be in lowercase
- Added validation to show the status ok or not using inserted data
- Added error message if the user inserts data out of the data submission rule
- Updated the contact form design to keep that it must be same as the Figma design

souad988 reviewed (approved):
Hi @yusufyusie , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Good PR description
DRY code

## Optional suggestions
_as suggested by the previous reviewer adding an onchange event listner to the inputs to hide error message when they change the email is good for user experience .

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
